src/mobu/models/business/sia.py

```diff
@@ -0,0 +1,54 @@
+"""Base models for SIA-related monkey business."""
+
+from __future__ import annotations
+
+from astropy.time import Time
+from pydantic import BaseModel, Field
+
+from .base import BusinessData, BusinessOptions
+
+__all__ = ["SIA2SearchParameters", "SIABusinessData", "SIABusinessOptions"]
+
+
+class SIA2SearchParameters(BaseModel):
```

Comment:
What do you think about `SIA2Query`? It seems like that's what it's representing when it's used

Reply:
I think that's what I had initially but then I gave the event the same name and wanted to avoid some confusion there. I changed it back to `SIA2Query`. In terms of the event that was actually `SIAQuery` which I'm now importing as an alias `SIAQueryEvent`. 

Would it make sense to also name that `SIA2Query`  for consistency? Or something like `SIA2QueryEvent`? Or perhaps just leave it as it is now?

Reply:
Ahh yeah I didn't think about the naming conflict with the event. I don't have a super strong opinion, whatever you think is best! For `SIA` vs `SIA2`, I'd say that should be the same in all the names, if I'm understanding correctly.

Reply:
...so if this is all for SIA v2, and we're not planning on having a v1 business, I'm totally fine with leaving the `2` off.

Reply:
Ok makes sense. Since I don't expect we'd be running an SIA v1 query I've removed all instances of the version number except for one place, which is the external pyvo `SIA2Service` class.